en/docs/consuming-services/create-an-application.md

```diff
@@ -1,51 +1,64 @@
-An application in Choreo is a logical representation of a physical application such as a mobile app, web app, device, etc. To consume an API in Choreo, you need to create an application that maps to your physical application and then subscribes to the required API over a usage policy plan that gives you a usage quota. A single application can have multiple subscriptions to APIs. Using the consumer key and consumer secret, you can generate an access token that you can use to invoke all the APIs subscribed to the same application.
+# Create an application in Choreo
 
-This section walks you through the steps to create an application in Choreo.
+An application in Choreo is a logical representation of a physical application, such as a mobile app, web app, or device. To consume an API in Choreo, you need to create an application that maps to your physical application and subscribe to the required API under a usage policy plan. This plan provides a usage quota. A single application can have multiple API subscriptions. Using the consumer key and consumer secret, you can generate an access token to invoke all APIs subscribed to the same application.
 
-Let's get started!
+This guide walks you through the steps to create an application in Choreo.
 
-### Step 1: Create an application
+## Step 1: Create an application
 
-To create an application in the Choreo Developer Portal, follow the steps given below:
+To create an application in the Choreo Developer Portal, follow these steps:
 
-1. Sign in to the [Choreo Developer Portal](https://devportal.choreo.dev).   
-2. Click **Applications**. and then click **+Create**.
-3. Enter a name and description for the application. 
-4. Click **Create**.
+1. **Sign in to the Choreo Developer Portal**  
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. Any reason to do this?